### ISSUE TITLE
Wifi Calling fails SP2A.220405.004.2022043000

### ISSUE THREAD

**Issue by solaslux** [GrapheneOS/os-issue-tracker#1159]
Hi there,

I have a pixel 6 pro on version 
SP2A.220405.004.2022043000
and WiFi calling no longer works. I use mint, a Tmobile mvno. I last successfully called over WiFi a few weeks ago.

The behavior is: I go to place a call, the phone appears to dial and then hangs up immediately. Turning off WiFi calling in the settings of the default phone app allows me to place calls over 5g as usual.

I have tried on two different WiFi networks. 
I have restarted my phone. 

My carrier settings version is 
tmomvno_us-33000000116.48

Any idea what might be the problem?

Thanks

**Comment by flawedworld** [comment]
Follow https://www.mintmobile.com/features/wifi-calling-text/

**Comment by solaslux** [comment]
For anyone else wondering, if I go to https://www.mintmobile.com/features/wifi-calling-text/ and check my IMEI, it will show that pixel 6 pro is listed as "not compatible" with Mint mobile's Wifi calling/texting. Save yourself a hassle and just disable it on the Mint account page and in the default phone app.

**Comment by solaslux** [comment]
> Make sure you have a SKU which was sold in the USA. Otherwise, they might just not support it yet.

I wanted to come back with the answers I've gotten. 

I've discussed my lack of WiFi calling with mint support and they confirmed that my imei shows as not supported. To reply to this, the phone was purchased in the USA from google directly. WiFi calling is supported by mint mobile on pixel 6 pros - but mine doesn't work. I have access to another pixel 6 pro with stock on the same provider and WiFi calling does work and the imei does show as compatible with the mint website imei checker.

Mint support had me switch to an eSIM from my physical sim to try to correct the issue but that didn't help. 

Even enabling all the required sandboxed gplay services to setup an eSIM did not allow me to use WiFi calling.

I am left with the assumption that either I've misconfigured something despite their support or grapheneOS is missing something that I am unable to identify in order to get it to work.
